lib/active_model/nested_attributes.rb

```diff
@@ -96,7 +96,11 @@ def define_validation_methods
             end
 
             false
-          end && super(*args, **kwargs, &block)
```

Comment:
Before, if the nested forms did not validate, it short-circuited the validation of the main form. Now we run both validations, storing results, and returning the `&&` of the results